http-servers/src/main/java/com/palantir/remoting/http/server/JerseyComponents.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.remoting.http.server;
+
+import io.dropwizard.setup.Environment;
+
+public final class JerseyComponents {
+
+    private JerseyComponents() {}
+
+    /**
+     * Registers standard exception mappers and message body writers.
+     *
+     * @param includeStackTrace
+     *        include the stack trace in exceptions written to the BODY of the http response; turn off when clients are
+     *        untrusted and shouldn't know details about the server implementation
+     */
+    public static void registerStandardComponents(Environment env, boolean includeStackTrace) {
```

Comment:
Should we be passing in whatever `env.jersey()` is instead of the `Environment`? We don't seem to be using anything from the top level object, and the class is named specifically for JerseyComponents.